### ISSUE TITLE
Shell in Australia (part 5000...)

### ISSUE THREAD

**Issue by lkernan** [osmlab/name-suggestion-index#9765]
It's time for the seemingly bi-annual Shell in Australia is rebranding again..

This time Coles Express is being removed and replaced with Reddy Express on the convenience stores.   
This is also a good time to revert to the standard Shell brand for the servo sites I believe as they are becoming proper Shell branding again (owned by Viva Australia but I don't think NSI needs to care about that distinction).

Later, some stores will change to OTR, but that brand is already established and already setup in NSI so easy fixed.

I haven't created a patch yet, but wanted to get others thoughts on:
1.  Removing the Coles Express fuel POI and replace with the Shell fuel POI.
2.  Renaming the Coles Express convenience POI to Reddy Express (the wikidata and wikipedia are already changed)

**Comment by 1ec5** [comment]
> owned by Viva Australia but I don't think NSI needs to care about that distinction

Right, this is something Wikidata would care about more, specifically in https://www.wikidata.org/wiki/Q110716465.

**Comment by Snowysauce** [comment]
This appears to have been completed:

https://nsi.guide/index.html?t=brands&k=amenity&v=fuel&tt=coles%20ex
https://nsi.guide/index.html?t=brands&k=shop&v=convenience&tt=coles%20ex
